_rules/document-title-is-descriptive.md

```diff
@@ -0,0 +1,198 @@
+---
+name: Document title is descriptive
+rule_type: atomic
+description: |
+  This rule checks that the document title describes the topic or purpose of the document.
+
+success_criterion: 
+- 2.4.2 # Page Titled (A)
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Corbb O’Connor 
+
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to the first HTML `title` element in a [document](https://www.w3.org/TR/dom/#concept-document) where the [document element](https://www.w3.org/TR/dom/#document-element) is an HTML `<html>` element, if the first HTML `title` element is [non-empty](#non-empty).
+
+### Expectation
+
+The target element describes the topic or purpose of the overall content of the [document](https://www.w3.org/TR/dom/#concept-document).
+
+## Assumptions
+
+- This rule assumes that browsers only recognize the first `title` element if multiple `title` elements are present in the [document](https://www.w3.org/TR/dom/#concept-document). Testing shows that this in general is the case. Therefore the scope of this rule is limited to only checking the first `title` element in a document.
```

Comment:
This seems more of an accessibility support note than an assumption. Assumptions are about the content you are testing, not about how well assistive technologies (which include user agents) conform to a standard.